docs/user/build-customization.rst

```diff
@@ -450,33 +450,41 @@ Take a look at the following example:
 Install dependencies with ``uv``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Projects can use `uv <https://github.com/astral-sh/uv/>`__,
-to install Python dependencies, usually reducing the time taken to install compared to pip.
-Take a look at the following example:
+Projects managed with `uv <https://github.com/astral-sh/uv/>`__, can install `uv` with asdf,
+and then rely on it to set up the environment and install the python project and its dependencies
+where readthedocs' own build steps expect it by setting the ``UV_PROJECT_ENVIRONMENT`` variable,
+usually reducing the time taken to install compared to pip.
```

Comment:
This is correct, but I think it would be valuable to break up into a few smaller sentences.